db/migrations/20181210092705-createFreckleAutomationTable.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+  up: (queryInterface, DataTypes) => queryInterface.createTable('freckleAutomation', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: DataTypes.INTEGER,
+    },
+    freckleUserId: {
+      type: DataTypes.STRING,
+      allowNull: true,
+    },
+    projectId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    status: {
+      type: DataTypes.ENUM,
+      values: ['success', 'failure'],
+      allowNull: false,
+    },
+    error: {
```

Comment:
@IAMOTZ , same here as statusMessage... If I am not wrong we might have a situation like "Success - User already allocated to project _xyz_"

Reply:
Yp, you are right.
I would update the column.